pkgs/development/libraries/nspr/latest.nix

```
@@ -0,0 +1,50 @@
+{ lib, stdenv, fetchurl
+, CoreServices ? null
```

```suggestion
, CoreServices
```

if this is null something is wrong.

---

Verbatim copy of the package from master. Also part of the stable package. I'd rather not change it in the port.